tests/nested/manual/uc-grub-boot-chains/modify-gadget.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import yaml
+
+parser = argparse.ArgumentParser()
+parser.add_argument('version_bump', type=int)
+parser.add_argument('type', choices=['removable', 'boot_entry'])
```

Comment:
this could use a comment to explain the two choices

Reply:
Done.